charts/cray-etcd-base/templates/migration-jobs.yaml

```diff
@@ -67,12 +67,39 @@ spec:
                chown -R 1001:1001 $snapshot_dir
                ns={{ .Release.Namespace }}
                cluster="{{ include "cray-etcd-base.fullname" . }}"
+               ss_name="${cluster}-bitnami-etcd"
 
-               kubectl get endpoints ${cluster}-etcd-client -n ${ns} -o json > /dev/null 2>&1
+               output=$(kubectl get endpoints ${cluster}-etcd-client -n ${ns} -o json > /dev/null 2>&1)
```

Comment:
Variable output doesn't seem to be used at all that I can tell and we're dumping stderr/out to /dev/null, think this is better to just be in the if like so:
```
if ! command ; then
```

Reply:
great comment -- fixed